changes/265-PrettyWood.md

```diff
@@ -0,0 +1,3 @@
+**Possible Breaking Change:** Inherited models as fields are not reconstructed (copied) anymore on validation
+To keep old behaviour, set `copy_on_model_validation = True` in the `Config` of your `BaseModel`.
+To change it globally for all your _pydantic_ models, you can set `BaseConfig.copy_on_model_validation = True`
```

Comment:
probably best to move some of this to the docs.

Reply:
Added the section from the other PR to explain how to change the behaviour globally.
Added the feature flag in the doc and change the default value to avoid any breaking change.
Ready for another review :)